reducers/plugins/index.ts

```diff
@@ -418,11 +418,11 @@ export default combineReducers({
     // components wrapped in an object that contains an id and plugin id
     components,
 
-    // object where every key is a post type and the values are components wrapped in an
+    // object where every key is a post type and the values are components wrapped in
```

Comment:
For this change and the one below, although they are helpful, it unnecessarily changes the git history of this file for the purpose of this PR. I suggest we revert this change so the history of file changes is more meaningful